modules/luci-mod-network/htdocs/luci-static/resources/view/network/interfaces.js

```diff
@@ -709,18 +709,22 @@ return view.extend({
 					so.depends('ra', 'hybrid');
 					so.depends('ra', 'relay');
 
-					so = ss.taboption('ipv6', form.ListValue, 'ra_management', _('DHCPv6-Mode'), _('Default is stateless + stateful<br />\
-						<ul style="list-style-type:none;">\
-						<li><strong>stateless</strong>: Router advertises prefixes, host uses <abbr title="Stateless Address Auto Config">SLAAC</abbr> \
-						to self assign its own address. No DHCPv6.</li>\
-						<li><strong>stateless + stateful</strong>: SLAAC. In addition, router assigns an IPv6 address to a host via DHCPv6.</li>\
-						<li><strong>stateful-only</strong>:  No SLAAC. Router assigns an IPv6 address to a host via DHCPv6.</li></ul>'));
-					so.value('0', _('stateless'));
-					so.value('1', _('stateless + stateful'));
-					so.value('2', _('stateful-only'));
+					so = ss.taboption('ipv6', form.Flag, 'ra_slaac', _('Enable <abbr title="Stateless Address Auto Config">SLAAC</abbr>'), _('Default is on.'));
+
+					so = ss.taboption('ipv6', form.MultiValue, 'ra_flags', _('RA Flags'), _('Default is managed config (M) and other \
+						config (O)<br /><ul style="list-style-type:none;">\
```

Comment:
If you suppress the list bullets via inline CSS I don't see a reason to use a list in the first place here (use of markup in translatable label texts complicates work for translators and substantially increases risk for potential errors). Maybe drop the markup entirely and simply use an english text enumeration?

Also, according to https://github.com/openwrt/odhcpd/blob/master/README#L117 the odhcpd default for this option is just `other-config`.

Reply:
I've tried to follow what I've seen done here https://github.com/openwrt/luci/blob/00d518d697fb37e40f405e7d9e7d6540f7243b0f/modules/luci-mod-network/htdocs/luci-static/resources/view/networks# L747, is it wrong?